src/group_theory/perm/cycles.lean

```diff
@@ -166,6 +166,18 @@ calc sign f = sign (swap x (f x) * (swap x (f x) * f)) :
         pow_one, units.neg_mul_neg] }
 using_well_founded {rel_tac := λ _ _, `[exact ⟨_, measure_wf (λ f, f.support.card)⟩]}
 
+lemma is_cycle_of_is_cycle_pow {σ : perm α} {n : ℤ}
+  (h1 : is_cycle (σ ^ n)) (h2 : σ.support ≤ (σ ^ n).support) : is_cycle σ :=
```

Comment:
Let's add a comment, e.g.
```
-- Here `h2` is equivalent to `σ.support = (σ ^ n).support`, but this formulation may be more usable
```
just so anyone coming across this is aware you've thought about it!